modules/account-lib/test/unit/keyPair/index.ts

```diff
@@ -1,13 +1,13 @@
 /**
  * @prettier
  */
-import 'should';
+import should from 'should';
 import { register } from '../../../src/keyPair';
 import * as coinModules from '../../../src';
 import { coins } from '@bitgo/statics';
 import { KeyPair as EthKeyPair } from '../../../src/coin/eth';
 import { KeyPair as Eth2KeyPair } from '../../../src/coin/eth2';
-import { KeyPair as DotKeyPair } from '../../../src/coin/dot';
+import { KeyPair as ed25519KeyPair } from '../../../src/coin/dot';
```

Comment:
I was trying to differentiate the key types based on the coins since we are testing based on different coins. It is fine to have this as the name if it is consistent across the board, e.g. BLSKeyPair, etc, but class name will need to be uppercase

Reply:
That was a mistake. Now is

https://github.com/BitGo/BitGoJS/blob/c647d6cb36a13e2fbccedde16b506f4eb3bbd1f7/modules/account-lib/test/unit/keyPair/index.ts#L10